W-19127798 Usage reports added usage alerts info jh (#335)
mulesoft/docs-general: modules/ROOT/pages/usage-reports.adoc

xref:rpa-home::ms-automation-credits-usage-types.adoc[Automation credits] for RPA, Composer, Intelligent Document Processing (IDP), Salesforce Flow Orchestration, and MuleSoft for Flow are sent to the primary billing contact.
+
For more information, see https://help.salesforce.com/s/articleView?id=000390885&type=1[Monthly Account Summaries] documentation in Salesforce.

* Alerts for usage reaching 80% and 90% of the contractual entitlement are sent automatically to the primary billing contact email address. The primary billing contact gets alerts monthly once thresholds are reached.
+

> For more information, see the https://help.salesforce.com/s/articleView?id=004811177&type=1[SalesforceUsage Alerts FAQ] knowledge article.

Reviewer: ```suggestion
For more information, see the https://help.salesforce.com/s/articleView?id=004811177&type=1[Salesforce Usage Alerts FAQ] knowledge article.
```

> For more information, see https://help.salesforce.com/s/articleView?id=004811177&type=1[SalesforceUsage Alerts FAQ] documentation in Salesforce.

Reviewer: ```suggestion
For more information, see the https://help.salesforce.com/s/articleView?id=004811177&type=1[SalesforceUsage Alerts FAQ] knowledge article.
```

Author: thank you
